Woof/WoofTests/Model/Preferences/PreferencesHandlerSetOwnerLoggedTests.swift

```diff
@@ -0,0 +1,26 @@
+import XCTest
+
+final class PreferencesHandlerSetOwnerLoggedTests: XCTestCase {
+    private var userPreferencesStorage = KeyValueStorage.testPreferencesStorage
+
+    override func setUp() {
+        super.setUp()
+        userPreferencesStorage.deleteData(for: KeyValueStorage.Key.userPreferences)
+    }
+
+    func testExistsInAPI() {
+        _ = PreferencesHandler.setOwnerLoggedOnce(true)
```

Comment:
nit
```suggestion
    func testMethodExistsInAPI() {
        _ = PreferencesHandler.setOwnerLoggedOnce(true)
```
You can keep your version or review the rest of code

Reply:
Thx fixed with 423791b